src/Common/src/Interop/Windows/kernel32/Interop.IoControlCodeAccess.cs

```diff
@@ -33,9 +33,7 @@ public enum IoControlCodeAccess
             /// </summary>
             FILE_WRITE_DATA = 0x02
 
-            /// <summary>
-            ///FILE_READ_DATA and FILE_WRITE_DATA can be ORed together if the caller must have both read and write access rights.
-            /// </summary>
+            // FILE_READ_DATA and FILE_WRITE_DATA can be ORed together if the caller must have both read and write access rights.
```

Comment:
Is this worth moving to the XML for those?

Reply:
I don't understand the question.  Can you clarify?  It was XML, but that causes problems because it wasn't associated with any valid target.

Reply:
Like this
```xml
            /// <summary>
            /// The I/O manager sends the IRP only for a caller with read access rights, allowing the underlying device driver to transfer
            /// data from the device to system memory.
            /// FILE_READ_DATA and FILE_WRITE_DATA can be ORed together if the caller must have both read and write access rights.
            /// </summary>
            FILE_READ_DATA = 0x01,

            /// <summary>
            /// The I/O manager sends the IRP only for a caller with write access rights, allowing the underlying device driver to transfer 
            /// data from system memory to its device.
            /// FILE_READ_DATA and FILE_WRITE_DATA can be ORed together if the caller must have both read and write access rights.
            /// </summary>
            FILE_WRITE_DATA = 0x02
```
Eh, it's probably too obvious to be worth mentioning.

Reply:
Ok. Thanks.  I'm going to leave it as-is.